Doc/library/dataclasses.rst

```diff
@@ -406,10 +406,10 @@ Module contents
          def add_one(self):
              return self.x + 1
 
-.. function:: replace(instance, /, **changes)
+.. function:: replace(obj, /, **changes)
```

Comment:
FTR the slash marks the arguments before it as positional-only, so the name chosen does not matter as it does not apply to code, it’s only for readers of the doc.  (But if all other instances are changed, it makes sense to change this one too.)

Reply:
Yeah this is more for consistency than anything else.

Reply:
The first argument for the other functions, `asdict()`, `astuple()`, and `isdataclass()`, is *not* positional-only. So chnaging the argument name is indeed a meaningful fix that should be made. In that context, indeed changing the name for `replace()` for consistency makes sense.